core/src/components/input/input.ionic.scss

```diff
@@ -176,8 +176,9 @@
 
 :host(.input-disabled) {
```

Comment:
The `--placeholder-color` needs to be changed as well. Design has neutral 500 when disabled.